main/course_info/about.php

```diff
@@ -124,7 +124,16 @@
 ];
 
 $subscriptionUser = CourseManager::is_user_subscribed_in_course($userId, $course->getCode());
-
+$course_table = Database::get_main_table(TABLE_MAIN_COURSE);
```

Comment:
You can use:

`$courseInfo = api_get_course_info_by_id($courseId);`

Then later you can check the subscribe value:

`$courseInfo['subscribe']`

Reply:
Awesome then it's even simpler to fix if this return a boolean.

I'd just be a assign and the same change in the template.